src/main/java/com/qa/app/Subtractor.java

```diff
@@ -0,0 +1,10 @@
+package com.qa.app;
+
+public class Subtractor {
+
+    public String Calculate(int number_one, int number_two) {
```

Comment:
Please change to camel case. For example, numberOne

Reply:
done